test/run-php-linter.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
For what it's worth, a dockerized .travis.yml would look like:
```
language: php

sudo: required

services:
  - docker

cache:
    directories:
        - vendor

env:
  - TEST_SUITE=lint:javascript
  - TEST_SUITE=lint:php
  - TEST_SUITE=tests:unit
  - TEST_SUITE=tests:integration

before_install:
    # Ensure node version is up to date
    - nvm install node
    - npm install
install:
    - composer install
script:
    - npm run --silent $TEST_SUITE
```

This breaks the tests up into 4 parts: unit tests, integration tests, PHP linting, and JS linting.